src/main/java/seedu/address/model/person/Person.java

```diff
@@ -97,7 +97,15 @@ public boolean isSamePerson(Person otherPerson) {
         }
 
         return otherPerson != null
-                && otherPerson.getName().equals(getName());
+                // && otherPerson.getName().equals(getName());
```

Comment:
Would it be better to remove this comment, if the line of code is no longer being used?